src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -134,6 +135,16 @@ public function validate($entity, Constraint $constraint)
         $errorPath = null !== $constraint->errorPath ? $constraint->errorPath : $fields[0];
         $invalidValue = isset($criteria[$errorPath]) ? $criteria[$errorPath] : $criteria[$fields[0]];
 
+        $vars = array();
+
+        if (preg_match_all('/{{ ([a-zA-Z0-9_]+) }}/i', $constraint->message, $vars) > 0) {
+            $accessor = PropertyAccess::createPropertyAccessor();
```

Comment:
You need to make sure that the PropertyAccess component is actually available and you should add it to the `suggest` section in `composer.json`.